src/SnoopCompile.jl

```diff
@@ -71,6 +71,51 @@ if VERSION >= v"1.2.0-DEV.573"
 
 end
 
+"""
+    timesum(snoop)
+
+Calculates and prints the total time measured by a snoop macro
+
+# Examples
+To benchmark the loadTime and RunTime of a package:
+- dev your package
+
+- comment the precompile part of your package (`include()` and `_precompile_()`)
+- run the following benchmark
+- restart Julia
+
+- uncomment the precompile part of your package (`include()` and `_precompile_()`)
+- run the following benchmark
+- restart Julia
+
+## Benchmark
+```julia
+using SnoopCompile
+
+println("Package load time:")
+loadSnoop = @snoopi using ColorTypes
+
+timesum(loadSnoop)
+
+println("Running Examples/Tests:")
+runSnoop = @snoopi begin
+    using ColorTypes
+    include(joinpath(dirname(dirname(pathof(ColorTypes))),"test","runtests.jl"))
+end
+
+timesum(runSnoop)
+```
+"""
+export timesum
+function timesum(snoop::Vector{Tuple{Float64, Core.MethodInstance}})
+    timeSum = 0
+    for x in snoop
+        timeSum+=x[1]
+    end
+    println(timeSum)
```

Comment:
```suggestion
```
If you return it, I don't think you have to print it. (The user can choose that for him/herself.)